### DISCUSSION TITLE
How do I not scale the canvas as a whole? Scale only the background image, not the inner rectangle

### DISCUSSION THREAD

**Discussion by zdsyyds** [fabricjs/fabric.js#7305] [Q&A]
![image](https://user-images.githubusercontent.com/88891895/129460140-225f33b8-596d-4007-8376-24be0f9ea3ed.png)
![image](https://user-images.githubusercontent.com/88891895/129460160-7675167e-5ee0-4038-aabf-4bdf1df3b396.png)

**Comment by asturur** [comment]
can you just apply scale to the background image?
That seems exactly what you want to do.
Or do you need the text boxes to mantain the relative position to the image while scaling?
